### PR TITLE
maintenance warning printed on STDERR rather than STDOUT

### LINKED ISSUE #24635: `occ app:list --output json_pretty` prints invalid JSON
### Steps to reproduce
1. occ maintenance:mode --on
2. occ app:list --output json_pretty
### Expected behaviour

Print only valid json to standard output.
### Actual behaviour

Print "ownCloud is in maintenance mode - no app have been loaded" to standard output before the JSON.
### Server configuration

**ownCloud version:** 8.2.4

### PR DESCRIPTION
**Pull request by noveens**
## Description

Fixes #24635 

If maintenance mode is enabled, the ```ownCloud is in maintenance mode - no app have been loaded``` warning is now printed in STDERR rather than STDOUT.

## Related Issue

https://github.com/owncloud/core/issues/24635

## How Has This Been Tested?

Tested by redirecting STDERR to a file, it shows me:
```ownCloud is in maintenance mode - no app have been loaded```

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

@PVince81

### CODE REVIEW

File: lib/private/Console/Application.php [hunk 1]
```diff
@@ -92,7 +93,8 @@ public function loadCommands(InputInterface $input, OutputInterface $output) {
 				if (\OCP\Util::needUpgrade()) {
 					throw new NeedsUpdateException();
 				} elseif ($this->config->getSystemValue('maintenance', false)) {
-					$output->writeln("ownCloud is in maintenance mode - no app have been loaded");
+					$errOutput = $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output;
```
Comment (PVince81) [inline lib/private/Console/Application.php]: can it be something else than `ConsoleOutputInterface`?

File: lib/private/Console/Application.php [hunk 2]
```diff
@@ -92,7 +93,8 @@ public function loadCommands(InputInterface $input, OutputInterface $output) {
...
+			        $errOutput->writeln('<comment>ownCloud is in maintenance mode - no app have been loaded</comment>' . PHP_EOL);
```
Comment (PVince81) [inline lib/private/Console/Application.php]: wrong indent

**Review by PVince81 — COMMENTED** [review]
yes for the general idea

File: lib/private/Console/Application.php (hunk 1 above)
Comment (noveens) [inline lib/private/Console/Application.php]: Any specific reasons?

Comment (PVince81) [inline lib/private/Console/Application.php]: I'm asking you about the reasons you chose to add this `instanceof` check

Comment (noveens) [inline lib/private/Console/Application.php]: I just introduced this check to check whether $output was already an instance of ConsoleOutputInterface, which it obviously isn't :P
I guess the check is redundant, i'll remove it.

Thanks.

**Comment by noveens** [comment]
@PVince81 ,
I have fixed the changes requested, can you review?

**Comment by PVince81** [comment]
Tested, works 👍

**Comment by noveens** [comment]
@PVince81 ,
merge?

**Pull request merged**